data/beps.yml

```diff
@@ -107,9 +115,10 @@
     link: https://github.com/bids-standard/bids-specification/pull/1128
     leads:
     -   name: Dejan Draschkow
+        email: dejan.draschow@psy.ox.ac.uk
```

Comment:
@DejanDraschkow This email seems dead. Are you still a BEP lead on BEP20? How should we contact you?